packages/mui-material/src/Autocomplete/Autocomplete.d.ts

```diff
@@ -273,7 +273,7 @@ export interface AutocompleteProps<
    */
   slotProps?: {
     clearIndicator?: Partial<IconButtonProps>;
-    paper?: PaperProps;
+    paper?: Partial<PaperProps>;
```

Comment:
This does not solve the issue. Please take a look at https://github.com/mui/material-ui/issues/38870#issuecomment-1751610970.